bin/templates/csr_access_test_template.S

```diff
@@ -0,0 +1,93 @@
+# CSR access test
```

Comment:
See the file's [history](https://github.com/openhwgroup/core-v-verif/commits/94bcce0f49e8896945cf0417740641bbe5f8f876/bin/templates/csr_access_test_template.S) for a list of my changes.